core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/network/NetworkV4Response.java

```diff
@@ -40,4 +40,13 @@ public Long getId() {
     public void setId(Long id) {
         this.id = id;
     }
+
+    @Override
+    public String toString() {
+        return "NetworkV4Response{" +
+                "id=" + id +
+                ", workspace=" + workspace +
```

Comment:
this information is not so useful. The id as well. Maybe did you forget to add the super toString?